source/reference/configuration.txt

```diff
@@ -0,0 +1,110 @@
+.. _c2c-config:
+
+Configuration File Settings
+###########################
+
+.. include:: /includes/preview-warning
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can configure :program:`mongosync` instances at startup using a configuration
+file.  The configuration file contains settings that are the equivalent of
+``mongosync`` command-line options.  See :ref:`c2c-config-setting-option-map`.
+
+
+Configuration File
+==================
+
+Most :program:`mongosync` command-line options can be written to file.
+The configuration file specifies values for each setting using YaML format.
+For example:
+
+.. code-block:: yaml
+
+   cluster0: "mongodb://192.0.2.10:27017"
+   cluster1: "mongodb://192.0.2.20:27017"
+   logPath: "/var/log/mongodb/mongosync"
+   verbosity: "WARN"
+
+To use a configuration file, start ``mongosync`` with the :option:`--config`
```

Comment:
Just curious if this connection needs to specified toward a `mongos` instance. If so might need to add mention of this.

Reply:
Reference was meant as generic to MongoDB, switched it to `/var/log/mongosync/` to avoid confusion.